src/core/namespace.c

```diff
@@ -496,6 +496,40 @@ static void drop_outside_root(const char *root_directory, MountEntry *m, unsigne
         *n = t - m;
 }
 
+static int clone_device_node(const char *d, const char *temporary_mount)
+{
```

Comment:
CODING_STYLE is to put the opening brace on the same line as the function declaration.  (Which I keep on forgetting).